charms/kfp-api/tests/integration/test_charm.py

```diff
@@ -86,6 +86,9 @@ async def test_relational_db_relation_with_mysql_relation(self, ops_test: OpsTes
         await ops_test.model.wait_for_idle(
             apps=["mysql-k8s"],
             status="active",
+            # mysql-k8s sometimes goes into error and then resolves itself.  Do not raise on error
```

Comment:
This change is already covered in #386, we can track this there.

Reply:
Yup I'll remove now that #386 went another direction